CodeConverter/CSharpToVB/ProgressReport.vb

```diff
@@ -0,0 +1,35 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Public Structure ProgressReport
+    Implements IEquatable(Of ProgressReport)
+
+    Public Sub New(Current As Integer, Maximum As Integer)
+        Me.Current = Current
+        Me.Maximum = Maximum
+    End Sub
+
+    Public ReadOnly Property Current As Integer
+    Public ReadOnly Property Maximum As Integer
+
+    Public Overrides Function Equals(obj As Object) As Boolean
+        Return (TypeOf obj Is ProgressReport) AndAlso Equals(DirectCast(obj, ProgressReport))
+    End Function
+
+    Public Overloads Function Equals(other As ProgressReport) As Boolean Implements IEquatable(Of ProgressReport).Equals
+        Return Current = other.Current AndAlso Maximum = other.Maximum
+    End Function
+
+    Public Overrides Function GetHashCode() As Integer
+        Return HashCode.Combine(Current, Maximum)
+    End Function
+
+    Public Shared Operator =(left As ProgressReport, right As ProgressReport) As Boolean
+        Return left.Equals(right)
+    End Operator
+
+    Public Shared Operator <>(left As ProgressReport, right As ProgressReport) As Boolean
+        Return Not left = right
+    End Operator
```

Comment:
I only added this because Fxcop Analyzers told me to. In my own projects I often suppress this rule. If you'd prefer to suppress here rather than add these methods, I'd be happy to do that.

Reply:
I like them but had some of the Roslyn tests fail with my implementation, I will try yours and can delete later.